.gitignore

```diff
@@ -9,6 +9,7 @@
 
 # dependencies
 /node_modules
+jspm_packages
```

Comment:
I'd like to know which part of the Task-2 does it satisfy? Why is this necessary to be added? Thank you!